diff --git i/PKGBUILD w/PKGBUILD
index 6ffd9ed..bef7d01 100644
--- i/PKGBUILD
+++ w/PKGBUILD
@@ -12,7 +12,7 @@
 
 _pkgname=resolve
 pkgname=davinci-resolve-studio
-pkgver=21.0.4
+pkgver=21.1
 pkgrel=1
 pkgdesc='Professional A/V post-production software suite from Blackmagic Design. Studio edition, requires license key or license dongle.'
 arch=('x86_64')
@@ -26,7 +26,7 @@ makedepends=('libarchive' 'xdg-user-dirs' 'patchelf')
 conflicts=('davinci-resolve' 'davinci-resolve-beta' 'davinci-resolve-studio-beta')
 source=("file://DaVinci_Resolve_Studio_${pkgver}_Linux.zip"
         "davinci-control-panels-setup.sh")
-sha256sums=('9fbbc5d919072d5d133e945d60e315b3d2acf3796732e3dac9dd3babb8b063cb'
+sha256sums=('06e9ac88060a95d81908f3ca334b0464e6d9a82761b4cb9654ad54aee802ff97'
             'f17236fd68cead727c647bc31404e402922cdd491df5526f4b62364cbef9d3b8')
 install="${pkgname}.install"
 options=('!strip')
@@ -84,14 +84,14 @@ prepare() {
     sed -i "s|RESOLVE_INSTALL_LOCATION|/opt/${_pkgname}|g" "${_file}"
   done < <(find . -type f '(' -name "*.desktop" -o -name "*.directory" -o -name "*.directory" -o -name "*.menu" ')' -print0)
 
-  rm squashfs-root/libs/libglib-2.0.so.0{,.6800.4} \
-     squashfs-root/libs/libgio-2.0.so.0{,.6800.4} \
-     squashfs-root/libs/libgmodule-2.0.so.0{,.6800.4} \
-     squashfs-root/libs/libgobject-2.0.so.0{,.6800.4} \
-     squashfs-root/libs/libc++.so.1{,.0} \
-     squashfs-root/libs/libc++abi.so.1{,.0}
-  ln -s "../BlackmagicRAWPlayer/BlackmagicRawAPI" "squashfs-root/bin/"
-  ln -s -t squashfs-root/libs/ \
+  rm -f squashfs-root/libs/libglib-2.0.so* \
+        squashfs-root/libs/libgio-2.0.so* \
+        squashfs-root/libs/libgmodule-2.0.so* \
+        squashfs-root/libs/libgobject-2.0.so* \
+        squashfs-root/libs/libc++.so* \
+        squashfs-root/libs/libc++abi.so*
+  ln -sf "../BlackmagicRAWPlayer/BlackmagicRawAPI" "squashfs-root/bin/"
+  ln -sf -t squashfs-root/libs/ \
     /usr/lib/lib{glib,gio,gmodule,gobject}-2.0.so.0 \
     /usr/lib/libgdk_pixbuf-2.0.so.0 \
     /usr/lib/lib{c++,c++abi}.so.1
